VA-Explorer/va_explorer
# Dashboard Responsivity Updates

This pull request:

- Moves the search box too its own line helping to frame the map. Adds search icon next to form element to increase prominence 
- Updates the layout to allow dashboard callout boxes content area to shrink/stack when screen width decreases.
- Updates the layout to allow dashboard charts and elements to stack when screen width decreases. Supported down to 480px width
- Links CoD metric up to all dashboard plots
- Bugfix: Incidentally fixes a small issue in the docker-compose config preventing `run_coding_algorithms` from connecting to pycrossVA open port.

Testing:
1. Login and navigate to the dashboard. 
2. Assuming chrome: Either use the developer console's device toolbar to simulate device resolutions or use an extension like [Window Resizer](https://chrome.google.com/webstore/detail/window-resizer/kkelicaakdanhinjdeammmilcgefonfh?hl=en) to change the window size to device presets.
3. Confirm that the dashboard elements respond to screen size changes accordingly down to 480px width.
4. Confirm that CoD dropdown at top of dashboard affects both dashboard plots when changed.

Closes: https://www.pivotaltracker.com/story/show/176870268
Closes: https://www.pivotaltracker.com/story/show/176517861
Closes: https://www.pivotaltracker.com/story/show/176869818
